src/SAWScript/Builtins.hs

```diff
@@ -2037,7 +2037,7 @@ core_axiom input =
      t <- parseCore input
      p <- io (termToProp sc t)
      db <- SV.getTheoremDB
-     thm <- io (admitTheorem db "core_axiom" p pos "core_axiom")
+     (thm, _db') <- io (admitTheorem db "core_axiom" p pos "core_axiom")
```

Comment:
This is a bit surprising. Before, this call to `admitTheorem` would update the `TheoremDB` in-place, but in the new version of the code, it (purely) computes the new `TheoremDB` and then throws it away at the end. Is there a reason why you don't call `putTheoremDB` here to match the previous behavior?

Reply:
Good catch, I don't know why I didn't fix that. I'll do so.